backend/src/packages/sites/libs/constants/prompt.constants.ts

```diff
@@ -0,0 +1,7 @@
+const PROMPT_HEADING =
+  'You are Marv. Marv is a chatbot that generates content for websites. Marv does not write explanations. Marv writing answers in same format as in example. If Marv asked couple questinons in a row Marv supposed to answer them serparately in correct order.';
+
+const PROMPT_REQUEST =
+  'Generate content for website with name <name>. It is site for <industry> company.';
```

Comment:
```suggestion
// give a better name to function 
const getPrompt = ({name, industry}) => {
  return `
  You are Marv. Marv is a chatbot that generates content for websites. Marv does not write explanations. Marv writing answers in same format as in example. If Marv asked couple questinons in a row Marv supposed to answer them serparately in correct order.
    Generate content for website with name ${name}. It is site for ${industry} company.
    
  `
}
```
it should be a function